tools/sigclient/functionalQueries/functionalQueries.yml

```diff
@@ -41,7 +42,6 @@ multi_value.json
 # inputlookup.json
 # inputlookup_2.json
```

Comment:
we should remove this two inputlookup cmds since they are at the top. also inputlookup3 can be uncommented ?

Reply:
I think @vasukalariya is removing them in a separate PR